src/GovUk.Education.ExploreEducationStatistics.Admin/Validators/ValidationMessages.cs

```diff
@@ -395,15 +395,6 @@ public static ErrorViewModel GenerateErrorDataSetTitleShouldNotContainSpecialCha
         Message: "Data set with title '{0}' cannot be replaced as it has an API data set linked to it. Please remove the API data set before replacing the data."
     );
 
-    public static ErrorViewModel GenerateErrorCannotReplaceDataSetWithApiDataSet(string title)
-    {
-        return new ErrorViewModel
-        {
-            Code = CannotReplaceDataSetWithApiDataSet.Code,
-            Message = string.Format(CannotReplaceDataSetWithApiDataSet.Message, title),
-        };
-    }
```

Comment:
I think this has left `LocalizableMessage CannotReplaceDataSetWithApiDataSet` unused now and it can also be removed.